api/publish.go

```diff
@@ -363,14 +452,27 @@ func apiPublishUpdateSwitch(c *gin.Context) {
 	})
 }
 
-// DELETE /publish/:prefix/:distribution
+// @Summary Delete published repository
+// @Description Delete a published repository.
```

Comment:
make bold: **Delete a published repository.**

copy api and field description from here: https://www.aptly.info/doc/api/publish/